diff --git a/node_modules/react-scripts/config/webpack.config.js b/node_modules/react-scripts/config/webpack.config.js
index 26c2a65..4d94724 100644
--- a/node_modules/react-scripts/config/webpack.config.js
+++ b/node_modules/react-scripts/config/webpack.config.js
@@ -132,17 +132,17 @@ module.exports = function (webpackEnv) {
           // https://github.com/facebook/create-react-app/issues/2677
           ident: 'postcss',
           plugins: () => [
-            require('postcss-flexbugs-fixes'),
-            require('postcss-preset-env')({
-              autoprefixer: {
-                flexbox: 'no-2009',
-              },
-              stage: 3,
-            }),
-            // Adds PostCSS Normalize as the reset css with default options,
-            // so that it honors browserslist config in package.json
-            // which in turn let's users customize the target behavior as per their needs.
-            postcssNormalize(),
+            // require('postcss-flexbugs-fixes'),
+            // require('postcss-preset-env')({
+            //   autoprefixer: {
+            //     flexbox: 'no-2009',
+            //   },
+            //   stage: 3,
+            // }),
+            // // Adds PostCSS Normalize as the reset css with default options,
+            // // so that it honors browserslist config in package.json
+            // // which in turn let's users customize the target behavior as per their needs.
+            // postcssNormalize(),
           ],
           sourceMap: isEnvProduction ? shouldUseSourceMap : isEnvDevelopment,
         },
